bears/ruby/RuboCopBear.py

```diff
@@ -28,8 +28,186 @@ def create_arguments(filename, file, config_file, rubocop_config: str=""):
+                        method_name: str='snake_case',
+                        string_literals: str='single_quotes',
+                        variable_name: str='snake_case',
+                        max_class_length: int=100,
+                        class_length_count_comments: bool=False,
+                        max_module_length: int=100,
+                        module_length_count_comments: bool=False,
+                        cyclomatic_complexity: int=6,
+                        max_line_length: int=80,
+                        line_length_allow_here_doc: bool=True,
+                        line_length_allow_uri: bool=True,
+                        max_method_length: int=10,
+                        method_count_comments: bool=False,
+                        max_parameters: int=5,
+                        count_keyword_args: bool=True,
+                        ignore_unused_block_args_if_empty: bool=True,
+                        allow_unused_block_keyword_arguments: bool=False,
+                        ignore_unused_method_args_if_empty: bool=True,
+                        allow_unused_method_keyword_args: bool=False):
+        """
+        Not all settings added.
+        Notable settings missing: Rails settings:
+
+        :param access_modifier_indentation:
+            Indent private/protected/public as deep as method definitions
+            options:
+                ``indent`` :  Indent modifiers like class members.
+                ``outdent`` : Indent modifiers one level less than
+                              class members.
+        :param preferred_alias:
+            Which method to use for aliasing in ruby?
+            options : ``alias`` , ``alias_method``.
+        :param align_hash_rocket_by:
+            Alignment of entries using hash rocket as separator.
+        :param align_colon_by:
+            Alignment of entries using colon as separator.
+        :param inspect_last_argument_hash:
+            Select whether hashes that are the last argument in a method call
+            should be inspected?
+            options: 'always_inspect', 'always_ignore', 'ignore_implicit',
+                     'ignore_explicit'
+        :param align_parameters:
+            Alignment of parameters in multi-line method calls.
+
+            The `with_first_parameter` style aligns the following lines along
+            the same column as the first parameter.
+
+            The `with_fixed_indentation` style aligns the following lines
+            with one level of indentation relative to the start of the line
+            with the method call.
+        :param class_check:
+            How to check type of class.
+            options: 'is_a?', 'kind_of?'
+        :param comment_keywords:
+            Checks formatting of special comments.
+        :param min_if_unless_lines:
+            Defines the number of lines of the body of an if / unless
+            needs to have to trigger this cop.
```

Comment:
this is still mixed up somehow, isn't it?

maybe "Defines the number of lines the body of an if/unless needs to trigger this cop."